ui/static/app/record/record.service.ts

```diff
@@ -14,25 +14,23 @@ export class RecordService {
   constructor(private http: HttpClient) { }
 
   getRecords(search: URLSearchParams): Observable<Response>  {
-    let url: string = "/api/records/";
+    let url: string = "/api/v2/records/";
```

Comment:
is it possible to make some "settings" where all those urls will be stored? you're reusing it (in this case `/api/v2/records/`) in so many places.. :)